src/Options/ContactDetails.php

```diff
@@ -0,0 +1,137 @@
+<?php namespace Leean\Elements\Options;
+
+/**
+ * Class ContactDetails.
+ *
+ * @package Leean\Elements\Options
+ */
+class ContactDetails
```

Comment:
Ok I see your point here.

However I think the social links are not ACF options we should use a menu for that instead since that works out of the box.

Let me know your thoughts on that. 

Thanks

> On Apr 4, 2016, at 6:51 AM, Caridad Zapatero Contreras <notifications@github.com> wrote:
> 
> In src/Options/ContactDetails.php <https://github.com/moxie-leean/wp-elements/pull/1#discussion_r58364037>:
> 
> > @@ -0,0 +1,137 @@
> > +<?php namespace Leean\Elements\Options;
> > +
> > +/**
> > + * Class ContactDetails.
> > + *
> > + * @package Leean\Elements\Options
> > + */
> > +class ContactDetails
> Hi @mitogh <https://github.com/mitogh>
> One of the collections of options is ContactDetails
> Another is SocialLinks
> 
> If I change the namespace, it doesn't make much sense to have Contact\Details and Social\Links.
> The are both collections of options and I think they should be in the same namespace.
> 
> I did try to change it anyway, but I couldnt get it to work either.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/moxie-leean/wp-elements/pull/1/files/9bec875f5d06bf504f566a44fc25ff74132c3416#r58364037>